src/AddIns/Uno.UI.Lottie/ts/Uno.UI.Lottie.ts

```diff
@@ -65,15 +68,19 @@ namespace Uno.UI {
 
 		public static pause(elementId: number): string {
 			this.withPlayer(p => {
-				this._runningAnimations[elementId].animation.pause();
+				const a = this._runningAnimations[elementId].animation;
+				a.pause();
+				this.raiseState(a);
 			});
 
 			return "ok";
 		}
 
 		public static resume(elementId: number): string {
 			this.withPlayer(p => {
-				this._runningAnimations[elementId].animation.play();
+				const a = this._runningAnimations[elementId].animation;
+				a.play();
+				this.raiseState(a);
```

Comment:
![Codacy](https://app.codacy.com/assets/images/favicon.png) Issue found: [Use the parent class name instead of `this` when in a `static` context.](https://app.codacy.com/gh/unoplatform/uno/pullRequest?prid=5436651)